Tasks/Common/securefiles-common/securefiles-common.ts

```diff
@@ -12,7 +13,19 @@ export class SecureFileHelpers {
         let serverCreds: string = tl.getEndpointAuthorizationParameter('SYSTEMVSSCONNECTION', 'ACCESSTOKEN', false);
         let authHandler = vsts.getPersonalAccessTokenHandler(serverCreds);
 
-        this.serverConnection = new vsts.WebApi(serverUrl, authHandler);
+        let proxy = tl.getHttpProxyConfiguration();
```

Comment:
nit: please use `const` instead of `let` (I know you were just following the pattern from above, but they should be `const` as well)